src/common/SurgeLogger.h

```diff
@@ -0,0 +1,10 @@
+/*
+ * SurgeLogger - quick wrapper to hold reference to and bootstrap the spd logger
+ */
+
+#ifndef __SURGELOGGER_INCLUDE__
```

Comment:
AFAICT, ifndef guards are used about six times in surge.  'pragma once' is used fifty-four.
I have nothing against ifndef guards, but I usually try to follow a project's conventions.

Reply:
Ha! OK so I had #pragma once for that reason, and @jsakkine suggested I replace it to be more standardized, so I did. Then I squashed my commits so you can't see that back and forth.

I think we need a clear style guide we can point to. Does it have pragma once or ifdef guards? I am completely neutral and will do either! But unfortunately we can't have both.

Reply:
If no other reason, at least ifndef cage is part of the standard whereas `#pragma once` is not. We are now IMHO defining the conventions (like the use of 4 space tabs, which is a relief :))

Reply:
Yeah I agree we need standards and perhaps another issue on them and how to get to them. Are we going to do a one day “super-retab”? Are we going to have mixed conventions in PRs? Do we have a tool which auto formats for us? Etc... All those things are great to discuss.

But I’d sooner not use the review of spdlog to resolve them :)

So can we decide whether, given the state of the code, these diffs are pushable? I’m fine either way of course! That’s why I asked for a review. But realizing that given the code is a bit here and there on tabs and spaces and names and ifdefs so we will incrementally improve, is this an incremental improvement? Especially now I’ve stripped out the stuff accidentally swept along as @whydoubt found and made the code way more consistent thanks to @jsakkine‘s eagle eyes!

Thanks!

Reply:
OK, lets do it that way! Lets have the fight about of space in an issue and then one atomic update to the code base :D